src/main/java/org/folio/rest/impl/DelayedTasksAPI.java

```diff
@@ -72,15 +85,15 @@ private Future<List<EmailEntity>> findEmailsForRetry() {
 
     return findEmailEntries(RETRY_BATCH_SIZE, 0, query)
       .map(EmailEntries::getEmailEntity)
-      .onSuccess(emails -> logger.info("Found {} emails for retry", emails.size()));
+      .onSuccess(emails -> logger.info("findEmailsForRetry:: Found {} emails for retry", emails.size()));
   }
 
   private static void logRetryResult(AsyncResult<Collection<EmailEntity>> result, long startTimeMillis) {
     long duration = currentTimeMillis() - startTimeMillis;
     if (result.succeeded()) {
-      logger.info("Email retry job took {} ms and finished successfully", duration);
+      logger.info("logRetryResult:: Email retry job took {} ms and finished successfully", duration);
     } else {
-      logger.error("Email retry job took {} ms and failed: {}", duration, result.cause().getMessage());
+      logger.error("logRetryResult:: Email retry job took {} ms and failed: {}", duration, result.cause());
```

Comment:
logger.warn? and there's extra placeholder

Reply:
updated